students.txt

```diff
@@ -7,7 +7,9 @@ List of students
 
 1. Anastasia P.
 2. Natalia T.
-3. Jorge T.
+3. George T.
+I like play volleyball, improve my front end developer skills, hangout with my friends...
```

Comment:
Please add more details of your personality. Favorite books, history persons, more activities, maybe career plans for next 5 years.